extensions/notifiers/ifttt.js

```diff
@@ -0,0 +1,27 @@
+module.exports = function container (get, set, clear) {
+  var ifttt = {
+    pushMessage: function(config, title, message) {
+      var request = require('request')
```

Comment:
Why you use the require inside the function? May be better do it before line 1, before `module.exports`.
Just my thinks. I know what V8 will optimise this, but it bad tone.

Reply:
I originally thought my IDE was telling me the require should be inside the function, vs outside of it, turned out I was just overwriting the var, leaving one unused (the actual error). I amended this now.